packages/db/src/services/DatabaseService.ts

```
@@ -339,3 +234,84 @@ export class DatabaseService {
+// TODO: Check every once in a while if Rfid and Timestamp can be made back into Uppercase
+export function flattenTelemetryData(
+  packet: ITelemetryData,
+): Partial<TelemetryPacket> {
+  const timestamp = new Date(packet.TimeStamp * 1000);
+  const Rfid = packet.Pi.Rfid;
+
+  return {
+    RaceName: packet.Title,
+    rfid: Rfid,
+    timestamp,
+    Title: packet.Title,
```

ESLint

[error] Expected object keys to be in ascending order. 'Title' should be before 'timestamp'.

(sort-keys/sort-keys-fix)